vividus-allure-adaptor/src/main/java/org/vividus/report/allure/plugin/DynamicPlugin.java

```diff
@@ -62,16 +65,22 @@ public DynamicPlugin(String pluginId, Supplier<List<String>> jsFileLinesSupplier
              */
             if (SystemUtils.IS_OS_UNIX)
             {
-                Files.setPosixFilePermissions(indexJs,
+                Files.setPosixFilePermissions(tempFile,
                         Set.of(PosixFilePermission.OWNER_WRITE, PosixFilePermission.OWNER_READ,
                                 PosixFilePermission.GROUP_READ, PosixFilePermission.OTHERS_READ));
             }
-
-            pluginFiles.put(INDEX_JS, Path.of(indexJs.toUri()));
-            getConfig().setJsFiles(List.of(INDEX_JS));
+            pluginFiles.put(fileToGenerate, Path.of(tempFile.toUri()));
+            Consumer<List<String>> fileConsumer = isJsFile(fileToGenerate)
+                    ? f -> getConfig().setJsFiles(f) : f -> getConfig().setCssFiles(f);
```

Comment:
it makes sense to be more restrictive: we need to fail if it's neither js nor css